extensions/swagger-ui/runtime/src/main/java/io/quarkus/swaggerui/runtime/SwaggerUiRecorder.java

```diff
@@ -26,9 +27,8 @@
             @Override
             public void handle(RoutingContext event) {
                 if (event.normalisedPath().length() == swaggerUiPath.length()) {
-
                     event.response().setStatusCode(302);
-                    event.response().headers().set(HttpHeaders.LOCATION, swaggerUiPath + "/");
+                    event.response().headers().set(HttpHeaders.LOCATION, swaggerUiPath + "/" + swaggerUiParams);
```

Comment:
If I understand correctly what you did, you only redirect with the parameters if someone went to `/swagger-ui` and then it's redirected to `/swagger-ui/?...options...`. What if the user goes to `/swagger-ui/` directly? Does it work?

Reply:
The response is: i tried to find a way to handle ```/swagger-ui/?``` but i failed. I need a help to find a way to fix this.

Reply:
Maybe the re-direction rule can be based on the presence or absence of configuration query params? If not, redirect using to the same request route with configuration query params. 

PS: This is a wild guess, I am not near my PC to check whether this could actually work.

Reply:
Hi, we should take in consideration the fact that the user could override this parameters at runtime. This is just a start-up configuration. 

```java
            Handler<RoutingContext> handler = recorder
                    .handler(SWAGGER_UI_FINAL_DESTINATION, httpRootPathBuildItem.adjustPath(swaggerUiConfig.path),
                            getSwaggerUiParams(swaggerUiConfig));
            routes.produce(new RouteBuildItem(swaggerUiConfig.path, handler));
            routes.produce(new RouteBuildItem(swaggerUiConfig.path + "/*", handler));
```
When i try to redirect ```swgger-ui/``` like this for example: 
```java
                } else if (event.normalisedPath().length() == swaggerUiPath.length() + 1) {
                    if (event.normalisedPath().endsWith("/")) {
                        event.response().setStatusCode(302);
                        event.response().headers().set(HttpHeaders.LOCATION, swaggerUiPath + swaggerUiParams);
                        event.response().end();
                        return;
                    }
                    event.reroute(swaggerUiPath + "/index.html");
                    return;
                }
```
Nothing happens. 

Notice that i haven't worked with Vert.x before.